gorp.go

```diff
@@ -257,17 +257,15 @@ func columnToFieldIndex(m *DbMap, t reflect.Type, name string, cols []string) ([
 			cArguments := strings.Split(field.Tag.Get("db"), ",")
 			fieldName = cArguments[0]
 
-			if fieldName == "-" {
-				return false
-			} else if fieldName == "" {
-				fieldName = field.Name
-			}
 			if tableMapped {
 				colMap := colMapOrNil(table, fieldName)
 				if colMap != nil {
 					fieldName = colMap.ColumnName
 				}
 			}
+			if fieldName == "" || fieldName == "-" {
```

Comment:
Were you able to ascertain the nature of `tableMapped` and what would happen on line 261 if `fieldName` hadn't been set to `field.Name`?

Reply:
Ya it supports the gorp `Rename` feature which we don't use. I don't believe this would break it even if we did. (but I have not tested it)